packages/date/src/DateRange.d.ts

```diff
@@ -15,6 +15,7 @@ export type DateRangeProps = {
   onChange?: (dates: { startDate: string; endDate: string }) => void;
   onPickerFocusChange?: (arg: { focusedInput: 'startDate' | 'endDate' | null }) => void;
   customArrowIcon?: React.ReactNode;
+  passThroughInvalidDates: boolean;
```

Comment:
We should mark `passThroughInvalidDates` as optional here

Reply:
also, what do you think about changing the prop to be called `allowInvalidDates`?

Reply:
do you think allowInvalidDates could be confusing as to whether it relates to form submission with invalid dates? I think its a close call - but the default behavior is counter-intuitive to me so my mental model here is likely off by a bit

Reply:
I think it could be confused with whether or not to allow people to even type invalid characters, eg: letters. 

The part that gives me pause is `passThrough`.  Maybe like `shouldSetInvalidDates`?